test/unit/test_runner_tests.rb

```diff
@@ -0,0 +1,97 @@
+require 'assert'
+require 'dk/test_runner'
+
+require 'dk/has_the_runs'
+require 'dk/runner'
+require 'dk/task'
+require 'dk/task_run'
+
+class Dk::TestRunner
+
+  class UnitTests < Assert::Context
+    desc "Dk::TestRunner"
+    setup do
+      @runner_class = Dk::TestRunner
+    end
+    subject{ @runner_class }
+
+    should "be a Dk::Runner" do
+      assert_true subject < Dk::Runner
+    end
+
+    should "have the runs" do
+      assert_includes Dk::HasTheRuns, subject
+    end
+
+  end
+
+  class InitTests < UnitTests
+    desc "when init"
+    setup do
+      @runner = @runner_class.new
+    end
+    subject{ @runner }
+
+  end
+
+  class RunTests < InitTests
+    desc "and run"
+    setup do
+      @params = { Factory.string => Factory. string }
```

Comment:
@kellyredding - there's an extra space after the `Factory.` and before `string` on the end